examples/tcp_rerouting/README.md

```diff
@@ -0,0 +1,81 @@
+## Proxy-Wasm plugin example: TCP Rerouting
+
+Proxy-Wasm TCP filter that dynamically routes connections to different upstream clusters based on the source IP address.
+
+Most WASM filter examples focus on HTTP, but this example shows how to work at the TCP/IP level.
```

Comment:
While this statement is true, it probably doesn't need to be permanently inscribed in the README.

Reply:
Adressed in https://github.com/proxy-wasm/proxy-wasm-rust-sdk/pull/332/commits/e68a1655a068cc4742c80ccafa453655fd50125c and https://github.com/proxy-wasm/proxy-wasm-rust-sdk/pull/332/commits/846ad4bd59d4a53c6487befefc95a6d981ec9da3